packages/data/src/index.js

```diff
@@ -24,14 +24,104 @@ export { createRegistrySelector, createRegistryControl } from './factory';
  * @param {Object} reducers An object whose values correspond to different reducing
  *                          functions that need to be combined into one.
  *
+ * @example
+ * ```js
+ * const { combineReducers, registerStore } = wp.data;
+ *
+ * const prices = ( state = {}, action ) => {
+ * 	return action.type === 'SET_PRICE' ?
+ * 		{
+ * 			...state,
+ * 			[ action.item ]: action.price,
+ * 		} :
+ * 		state;
+ * };
+ *
+ * const discountPercent = ( state = 0, action ) => {
+ * 	return action.type === 'START_SALE' ?
+ * 		action.discountPercent :
+ * 		state;
+ * };
+ *
+ * registerStore( 'my-shop', {
+ * 	reducer: combineReducers( {
+ * 		prices,
+ * 		discountPercent,
+ * 	} ),
+ * } );
+ * ```
+ *
  * @return {Function}       A reducer that invokes every reducer inside the reducers
  *                          object, and constructs a state object with the same shape.
  */
 export { combineReducers };
 
+/**
+ * Given the name of a registered store, returns an object of the store's selectors. The selector functions are been pre-bound to pass the current state automatically. As a consumer, you need only pass arguments of the selector, if applicable.
```

Comment:
Can we cap the line length here (and in other functions)? I'd suggest splitting it across 80 columns lines.

Reply:
Done at 153a6b4